indico/modules/users/controllers.py

```diff
@@ -303,15 +312,69 @@ def _process(self):
                         **{k: entry.data.get(k) for k in (IDENTITY_ATTRIBUTES - {'full_name'})}))
 
             search_results.sort(key=attrgetter('first_name', 'last_name'))
-        return WPUsersAdmin.render_template('users_admin.html', form=form, search_results=search_results,
-                                            num_of_users=num_of_users, num_deleted_users=num_deleted_users)
+
+        if self._user_management_form_posted() and user_management_form.validate_on_submit():
+            self._save_account_mgmt_settings(user_mgt_form_data)
+
+        return WPUsersAdmin.render_template('users_admin.html', form=search_form, search_results=search_results,
+                                            num_of_users=num_of_users, num_deleted_users=num_deleted_users,
+                                            user_management_form=user_management_form)
+
+    def _create_user_management_form(self):
+        return UserManagementForm(obj=FormDefaults(**self._load_management_settings()))
+
+    def _load_management_settings(self):
+        info = HelperMaKaCInfo.getMaKaCInfoInstance()
+        return {
+            'notify_on_new_account': info.getNotifyAccountCreation(),
+            'local_account_creation': info.getAuthorisedAccountCreation(),
+            'account_moderation_workflow': info.getModerateAccountCreation()
+        }
+
+    def _user_management_form_posted(self):
+        return 'save_management_form' in self.getRequestParams()
+
+    def _save_account_mgmt_settings(self, data):
+        info = HelperMaKaCInfo.getMaKaCInfoInstance()
+        info.setNotifyAccountCreation(data.pop('notify_on_new_account'))
+        info.setAuthorisedAccountCreation(data.pop('local_account_creation'))
+        info.setModerateAccountCreation(data.pop('account_moderation_workflow'))
+        transaction.commit()
 
 
 class RHUsersAdminCreate(RHAdminBase):
     """Create user (admin)"""
 
     def _process(self):
+        # to avoid circular imports
+        from indico.modules.auth.controllers import LocalRegistrationHandler
+
         form = LocalRegistrationForm()
+        data = form.data
+        handler = LocalRegistrationHandler(self)
+
+        if form.validate_on_submit():
```

Comment:
this block looke a LOT like duplicated code. can't we add a `create_user` util instead?